lib/internal/modules/ts/typescript.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const { init, transformSync } = require('internal/deps/swc/wasm-web');
+const { readFile } = require('fs/promises');
+const { pathToFileURL } = require('internal/url');
+
+async function load(url, context, nextLoad) {
+  await init();
+  const buffer = await readFile(pathToFileURL(url));
+  const { code } = transformSync(buffer.toString(), {
+    jsc: {
```

Comment:
```suggestion
    swcrc: false,
    jsc: {
```

sets this property to ensure that node doesn't search for `.swcrc`.